model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
```

Comment:
Do we want to record a different timestamp for each app that is opened (so if the app has been opened 100 000 we record 100 000 timestamps) ? We can, we should change the model then